### DISCUSSION TITLE
Look and Feel UI/UX improvements – Better integrations of Zen color scheme & utility themes into the settings, and more

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#797] [Ideas]
### What feature would you like to see?

![Zentheming Suggestion 1_pages-to-jpg-0001](https://github.com/user-attachments/assets/d873a12e-04a3-4ca4-8d36-5ee7d98b687f)

### 1. **Move appearance menu from _Language and Appearance_ section (_General_) to _Look and Feel_ section (_Look and Feel_)**

- a) **Changing “_Website appearance_” (automatic/light/dark mode switch) to “_Overall appearance_”** to make it more relevant - applies not only to websites, but also to the browser itself.
- We got to introduce that **modern themes will supposed to accomodate light and dark mode simultaneously in one package**, compared to Firefox add-on themes that only use one mode.
----
- b) In order to apply that, I suggest that we should **separate Zen-owned themes** (_Dark-Light_, _Zen Galaxy-Zen Dream_) **from Firefox add-on themes and convert it as “theming mode”**.
- When using **_Flat_** (_Dark/Light_) or **_Dreamy_** (_Zen Galaxy/Zen Dream_) theming mode, only Zen color schemes will be used.
- Otherwise, when _Custom_ theming mode is used, browser theming (the top URLbar, background in Settings/Extensions/Firefox View will be completely depend on Firefox add-on themes you currently use. (Example: If you applied Teal color scheme on Zen's _Look and Feel_ settings, those color won’t apply on the browser elements or menus.)
----
- c) (Optional) **Move _Colors_, _Fonts_, _Zoom_ menu and put it into a separate category in _Look and Feel_ settings** (probably above Side Web Panels menu)

---
	
### 2. Renaming “_Pick a theme colour_” to “_Pick a theme color_” for more cohesive copywriting 
(like, with “_accent color_” on the same menu, or “_color-scheme_” on the website)

---

### 3. Improve _Pick a theme color_ menu & integrate user-made color schemes 

- a) **Put labels to each color-scheme** (so we can differentiate default vs user-made color schemes).
- b) **Adding + (plus) button to install user-made color scheme**, which opens the [_Theme Store_](https://www.zen-browser.app/themes) and auto-filter ‘_color-scheme_’ themes.
- c) **Integrate user-made color schemes** alongside default color schemes.

![Zentheming Suggestion 1_pages-to-jpg-0004](https://github.com/user-attachments/assets/079ea744-3dbc-49c6-ba1a-3a0cd876d628)

(the _Show in compact view_ will be located below _Theming mode_ menu)

---

### 4. Renaming "_Theme Settings_" section to "_Utility Settings_"

to make clearer distinction between changing the color schemes (F1, Catpucinno... these belongs to _Look and Feel_/_color scheme_ menu above) vs tweaking utility side of Zen Browser itself (Hide Window Buttons, Compact Sidebar, Bookmark Toolbar Tweaks etc... these belongs to _Utility Settings_ here.)

---

### 5. Giving better visual distinction between '_color-schemes_' and '_utility_' themes in the [_Theme Store_](https://www.zen-browser.app/themes/)

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/569_

**Comment by realSilasYang** [comment]
A comparison between Firefox and Zen shows there's room for improvement in dark mode optimization. Zen should adjust the black background to a true dark scheme color.
![myznpmex rkx](https://github.com/user-attachments/assets/16d078f4-4fbd-4afd-bbb0-20dbdcdd0ed2)
![4c1pw0v1 xmy](https://github.com/user-attachments/assets/0d3b4001-98c6-4d45-b179-91dab97440a4)
![yhvwnphu jru](https://github.com/user-attachments/assets/e0ba7ad8-60cd-41e7-bdf9-2e64b8f87828)
![x2q1htua rld](https://github.com/user-attachments/assets/5aa6dc31-4d1b-42b2-9788-74e51374ffa6)
